packages/extension-chakra-storefront/src/pages/login/index.test.js

```diff
@@ -5,8 +5,7 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 import React from 'react'
-import {screen, waitFor} from '@testing-library/react'
-import {rest} from 'msw'
+import {act, screen, waitFor} from '@testing-library/react'
```

Comment:
I assume act has been added only for the tests that has warning of falky?

Reply:
yes I wrapped them all with act. So far I've noticed no flakiness, which is a good sign.